frontend/src/schemas/SchemaSelect.tsx

```diff
@@ -153,7 +153,7 @@ export default function SchemaSelect({ entry }: SchemaSelectProps) {
               </Accordion>
               <Accordion sx={accordionStyling}>
                 <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-                  <Typography sx={{ width: '100%' }} align='center' color='primary' variant='h6' component='h2'>
```

Comment:
Also consider reverting this. The codemod hasn't even added slotProps like the above accordion, so it's definitely not right.